### ISSUE TITLE
Threat

### ISSUE THREAD

**Issue by SpaceDiceMan** [Stefouch/myz-zonemap#10]
There seems enough space to also provide full description of threats (would save page-turning time).

**Comment by Stefouch** [comment]
This is a feature I've planned to do in a near future. I'm already, piece by piece, encoding the threats book chapter in a database. :)

**Comment by SpaceDiceMan** [comment]
Those are already available from other Zone Generators

**Comment by lukbeg** [comment]
Threats draw only one 1. Why they are not displayed more than in the case of artifacts

**Comment by Stefouch** [comment]
@lukbeg According to the rules pg. 158, only 1 threat is drawn:

> If you roll at least one ☣, there is a threat in the sector, and the more ☣ you roll, the more dangerous the threat is.

**Comment by lukbeg** [comment]
I agree but Find My Path: it doesn't show me more then 1 threat no matter how many rolls I do, only 1 threat so its a bug yes?
